test/Autofac.Test/Core/Activators/ProvidedInstance/ProvidedInstanceActivatorTests.cs

```diff
@@ -20,9 +20,9 @@ public void NullIsNotAValidInstance()
         [Fact]
         public void WhenInitializedWithInstance_ThatInstanceIsReturnedFromActivateInstance()
         {
-            object instance = new object();
+            object instance = new();
 
-            ProvidedInstanceActivator target = new ProvidedInstanceActivator(instance);
+            ProvidedInstanceActivator target = new(instance);
```

Comment:
For method fields, I'd usually lean towards using `var` and leaving the type on the other side rather than put the type at the front this way...possibly personal preference.